jetty-alpn/jetty-alpn-conscrypt-server/src/test/resources/jetty-logging.properties

```diff
@@ -1,3 +1,3 @@
 org.eclipse.jetty.util.log.class=org.eclipse.jetty.util.log.StdErrLog
-#org.eclipse.jetty.LEVEL=DEBUG
-#org.eclipse.jetty.alpn.LEVEL=DEBUG
+org.eclipse.jetty.LEVEL=DEBUG
+org.eclipse.jetty.alpn.LEVEL=DEBUG
```

Comment:
These should be commented out.

Reply:
I prefer leave this ATM (at least until conscrypt issue is fixed) otherwise the conscrypt exception is not displayed in the logs..
It's used to help conscrypt folks see related issue here https://github.com/google/conscrypt/issues/635